manywheel/build_libtorch.sh

```diff
@@ -115,6 +115,11 @@ else
     export _GLIBCXX_USE_CXX11_ABI=0
 fi
 
+if [[ "$DESIRED_CUDA" == *"rocm"* ]]; then
+    echo "Calling build_amd.py at $(date)"
+    python tools/amd_build/build_amd.py
```

Comment:
Do we need to exit after calling build_amd.py? Or continue execution?

Reply:
Continue execution, yes.  This is the step that hipifies the pytorch sources.